Rivet/Functions/Connect-Database.ps1

```diff
@@ -45,29 +45,36 @@ function Connect-Database
         Disconnect-Database -Session $Session

```

Comment:
Why do we call `Disconnect-Database` if there isn't a connection or if the connection is already Closed?

Reply:
So there's only one place we're calling `connection.Close()`.